src/back/TicketManager/src/PaymentService/TicketManager.PaymentService.Services/Endpoints/Payments/CheckPaymentStatusValidator.cs

```diff
@@ -0,0 +1,35 @@
+﻿using FastEndpoints;
+using FluentValidation;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using TicketManager.PaymentService.Contracts.Payments;
+using TicketManager.PaymentService.Services.Extensions;
+using TicketManager.PaymentService.Services.Services.Mockables;
+
+namespace TicketManager.PaymentService.Services.Endpoints.Payments;
+
+public class CheckPaymentStatusValidator: Validator<CheckPaymentStatusRequest>
```

Comment:
Generally we don't validate get requests, because they don't change anything in databse and if you get bad results based on bad request, it's you problem. But if it's already here, it can stay.